docs/diagrams/WhoIsFreeActivityDiagram.puml

```diff
@@ -2,7 +2,9 @@
 start
 :User executes command;
```

Comment:
I think previously @vishandi also mentioned this, the activity diagram doesn't really add much value to the DG, I think it would be better to not include it

Reply:
ya, but in the interest of time, I think if we could not resolve this just yet it is fine, we could do the documentations next week. We need to make the functionalities work well first imo.

Reply:
Yup okay